src/main/javahelp/help/contents/intro.html

```diff
@@ -8,18 +8,24 @@
 </HEAD>
 <BODY>
 <H1>Neonmarker</H1>
+<p>
 Neonmarker addon, which colours History table entries based on tags.
-
+</p>
+<p>
 The add-on also facilitates colouring arbitrary messages in the History table via a right-click context menu. Tags are added to the selected messages to deliver the necessary colouring.
 Neonmarker's custom tags take the form <code>neon_UUID</code>, for example: <code>neon_e8b1d1e6-9dd4-4996-bc02-de2213986352</code>. Another context menu item provides functionality to 
 remove Neonmarker's custom tags.
-
+</p>
+<p>
 The following code example shows how to add a Neonmarker colorMapping via JavaScript within ZAP using a Stand Alone script.
-
-<code>extNeon = org.parosproxy.paros.control.Control.getSingleton().getExtensionLoader().getExtension(org.zaproxy.zap.extension.neonmarker.ExtensionNeonmarker.NAME);
+</p>
+<p>
+<code>
```

Comment:
I think it needs to use `<pre>`.

Reply:
Oh, pre then code or code then pre? (Or doesn't matter as long as both are in play)

Reply:
Found other instances with pre first. Will tweak.